tooltester/src/main/resources/setupSlave.sh

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
-# Can't actually run this entire script (there's interactive elements), but it gives a general idea what needs to be done to setup slave
```

Comment:
Out of curiosity, what changed here to presumably allow this to be run non-interactively?

Reply:
I moved the interactive parts out